app/model/StateHistory.scala

```diff
@@ -22,6 +22,9 @@ import org.joda.time.DateTime
 import play.api.libs.json.Json
 import model.State.State
 import uk.gov.hmrc.time.DateTimeUtils
+  //TODO: Is the default format for Joda correct for Gatekeeper?
+  import play.api.libs.json.JodaReads._
+  import play.api.libs.json.JodaWrites._
```

Comment:
Anybody got views on this?
Not sure just using Joda implicits will be correct in this context.